lib/jest-helpers.js

```diff
@@ -15,38 +15,57 @@ const sassRender = util.promisify(sass.render)
 const configPaths = require('../config/paths.json')
 const { componentNameToMacroName } = require('./helper-functions.js')
 
-nunjucks.configure([configPaths.src, configPaths.components], {
+const nunjucksEnv = nunjucks.configure([configPaths.src, configPaths.components], {
   trimBlocks: true,
   lstripBlocks: true
 })
 
 /**
- * Render a component's macro for testing
- * @param {string} componentName
- * @param {string} params parameters that are used in the component macro
- * @param {any} children any child components or text, pass the children to the macro
- * @returns {function} returns cheerio (jQuery) instance of the macro for easy DOM querying
+ * Render the raw HTML for a component
+ *
+ * @param {String} componentName
+ * @param {Object} options options to pass to the component macro
+ * @param {String} callBlock if provided, the macro is called using the Nunjucks
```

Comment:
Not sure how tight we want to be on that and maybe a bit pedantic, but thought we may as well [show the parameter is optional in the JSDoc](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values).

```suggestion
 * @param {String} [callBlock] if provided, the macro is called using the Nunjucks
```